nasa-gibs/worldview
# Tour step location change fix

## Description

Fixes #3418.

I'm not entirely sure why this error was suddenly happening.  For some reason, `payload.query` was undefined in the `LOCATION_POP_ACTION` reducer in `map/ui.js`.  In this instance, `payload` is actually the `location` object which comes from the `history` package and is imported into `web/js/containers/tour.js`.  

This code hasn't changed for several versions even though it appears to work in the currently deployed PROD but does not work locally for `develop` or `main`.  Neither the `history` nor the `redux-location-state` packages have been updated recently.

Since `location.query` was essentially what was on the current Redux state, I updated this to pull directly from `state` instead which seems to work fine.  I'm not sure if this should be included in 3.9.1 or 3.10.0 since it appears to work in PROD 🤷🏻‍♂️  

## How To Test

1. Open Worldview
2. Load Australia bushfires story
3. Step through and confirm map extent location updates properly as you work through the story.  Compare against PROD if necessary to confirm.
